common/src/main/java/com/wynntils/core/consumers/overlays/TextOverlay.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © Wynntils 2023.
```

Comment:
Did you change this yourself?

Reply:
It should be 2023-2024.

Reply:
Oh, is it changed automatically? I had no clue, so I changed it myself

Reply:
Yes. It's done by a formatting bot on GH, or the git hook the README mentions. Either way, no need to touch it. We are lazy programmers :)